src/Matterhorn/State/Messages.hs

```diff
@@ -681,6 +686,57 @@ runNotifyCommand post mentioned = do
                                  [notified, sender, messageString] Nothing Nothing
                 return Nothing
 
+encodeToJSONstring :: A.ToJSON a => a -> String
+encodeToJSONstring a = map (chr . fromEnum) . BL.unpack $ A.encode a
+
+-- We define a notifyGetPayload for each notification version.
+notifyGetPayload :: Int -> ChatState -> Post -> Bool -> MH (Maybe String)
+
+-- Notification Version 2
+data NotificationV2 = NotificationV2
+    { version :: Int
+    , message :: Text
+    , mentioned :: String
+    , sender :: Text
+    } deriving (Show)
+
+instance A.ToJSON NotificationV2 where
+    toJSON ( NotificationV2 version message mentioned sender ) =
```

Comment:
Style note: no whitespace around the pattern-matched constructor here. (So just `(NotificationV2 ... sender)`).

Reply:
Resolved by commit 54d3635c .

Reply:
Resolved by commit 54d3635 .